### PR TITLE
Updates contributing guide to discuss the code copy button usage

### PR DESCRIPTION
**Pull request by emanic**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: CONTRIBUTING_DOCS.md [hunk 1]
```diff
@@ -204,11 +204,21 @@ An anchor link for each heading is automatically created. It consists of the tit
...
 
-Each page should include a `canonical_url` tag that contains the absolute path to the latest copy of the page, even if the latest copy is the page itself. 
+Each page should include a `canonical_url` tag that contains the absolute path to the latest copy of the page, even if the latest copy is the page itself.
 
 You should _not_ need to modify the `canonical_url` metadata unless you are adding, deleting, or renaming a page.
 
+## Code samples
+
+Our site adds a copy button to each code block by default. To ensure that readers can copy and paste the code successfully, follow the [Code samples](https://github.com/emanic/calico/blob/contr-update/DOC_STYLE_GUIDE.md#code-samples) recommendations in the DOC_STYLE_GUIDE.
```
Comment (caseydavenport) [inline CONTRIBUTING_DOCS.md]: We shouldn't link to your personal fork here for the style guide.

Comment (emanic) [reply CONTRIBUTING_DOCS.md]: Oh, oops.

File: CONTRIBUTING_DOCS.md [hunk 2]
```diff
@@ -204,11 +204,21 @@ An anchor link for each heading is automatically created. It consists of the tit
...
 
-Each page should include a `canonical_url` tag that contains the absolute path to the latest copy of the page, even if the latest copy is the page itself. 
+Each page should include a `canonical_url` tag that contains the absolute path to the latest copy of the page, even if the latest copy is the page itself.
 
 You should _not_ need to modify the `canonical_url` metadata unless you are adding, deleting, or renaming a page.
 
+## Code samples
+
+Our site adds a copy button to each code block by default. To ensure that readers can copy and paste the code successfully, follow the [Code samples](https://github.com/projectcalico/calico/blob/contr-update/DOC_STYLE_GUIDE.md#code-samples) recommendations in the DOC_STYLE_GUIDE.
```
Comment (caseydavenport) [inline CONTRIBUTING_DOCS.md]: @emanic this link doesn't seem to exist.

I think you want this: https://github.com/projectcalico/calico/blob/master/DOC_STYLE_GUIDE.md#code-samples

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 9bf4925811005191e452f45328a4aeb290378f87

https://deploy-preview-2274--calico.netlify.com

**Pull request merged**